stable/redis-ha/templates/redis-sentinel-service.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     name: {{ template "name" . }}-sentinel-svc
     role: service
 {{ include "labels.standard" . | indent 4 }}
-  name: redis-sentinel
+  name: {{ .Release.Name }}-redis-sentinel
```

Comment:
The correct name should use the `fullname` template.

`{{ template "fullname" . }}-redis-sentinel`

For some reason, some of the chart's resources also have name labels which don't match the resource name. Seems weird to me.